hibernate-reactive-core/src/test/java/org/hibernate/reactive/MultipleContextTest.java

```diff
@@ -0,0 +1,146 @@
+/* Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ * Copyright: Red Hat Inc. and Hibernate Authors
+ */
+package org.hibernate.reactive;
+
+import java.util.concurrent.CompletionException;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.mutiny.Mutiny;
+import org.hibernate.reactive.stage.Stage;
+
+import org.junit.Test;
+
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+import io.vertx.ext.unit.TestContext;
+import org.assertj.core.api.Assertions;
+
+/**
+ * It's currently considered an error to share a Session between multiple reactive streams,
+ * so we should detect that condition and throw an exception.
+ */
+public class MultipleContextTest extends BaseReactiveTest {
+
+	private static final String ERROR_MESSAGE = "Detected use of the Reactive Session from a different Thread than the one which was used to open the Reactive Session - this suggests an invalid integration;";
```

Comment:
What's the ; for?

Reply:
Also, "Reactive" should be lowercase, since it's not a type or a proper noun.

Reply:
The error message is longer than the string I use here and contains a semicolon. I've update the assert to check a shorter version of the string.

I've also replaced 'Reactive' with 'reactive'

Reply:
Thanks!